README.md

```diff
@@ -42,8 +42,8 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
-```
+for(var i = 0; i < foods.length; i++){
+      console.log("I want to eat "+foods[i]+"")
```

Comment:
Doesn't look like you closed this loop properly. Also, note that there are arrays within the larger `foods` array. That means you'll need to nest a loop in your existing loop in order to print out each fruit individually.